atlasdb-config/src/main/java/com/palantir/atlasdb/http/errors/AtlasDbRemoteException.java

```diff
@@ -45,6 +51,19 @@ public String getErrorName() {
         return remoteException.getRemoteException().getErrorName();
     }
 
+    // Cannot be named getStackTrace() because that returns a list of (non-serializable) StackTraceElements.
```

Comment:
They both take 0 arguments and have the same name, so they have the same signature - [JLS 8.4.2](https://docs.oracle.com/javase/specs/jls/se8/html/jls-8.html#jls-8.4.2).

(Notice the return type isn't part of the comparison)